### ISSUE TITLE
Not able to set password after user created

### ISSUE THREAD

**Issue by santoshshelar27** [ldapjs/node-ldapjs#954]
Hi Team, 
I have a requirement to create a user and set its default password using Node.js. I'm encountering an issue while attempting to set the password after creating the user. Both the "Add" and "Replace" operations result in the following error:

LDAP modify failed: LDAPError [InsufficientAccessRightsError]: Insufficient Access Rights

**Key points to note:**

1. I am using an admin user for these operations.
2. The same user credentials work correctly in a .NET application.
3. I have referred to this link for additional context: [link](https://github.com/ldapjs/node-ldapjs/blob/a433489e982bcbfcd0c6f8d7ed985ef236569284/test-integration/client/issue-946.test.js#L70)

` const change = new ldap.Change({
     operation: 'replace',
     modification: new ldap.Attribute({
         type: 'userPassword',
         values: password
     })
 });
 client.modify(userDN, change, (err, res) => {
     if (err)
         console.error('LDAP modify failed:', err);
     console.log("Modify done.")

 });`

Your assistance in resolving this issue would be highly appreciated. If there's any additional information needed, please let me know.

**Comment by mmjayc** [comment]
Does the password you are trying to set, meet your n password policy?

**Comment by santoshshelar27** [comment]
Yes, the password we are attempting to set does meet the LDAP password policy.

It's worth noting that the same set of details allows us to successfully create a user on LDAP using a .NET application without encountering any issues. Therefore, we believe the password adheres to the LDAP password policy.

**Comment by jsumners** [comment]
Please provide a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example) (MRE). Doing so will help us diagnose your issue. It should be the bare minimum code needed to trigger the issue, and easily runnable without any changes or extra code. Please review the [integration tests](https://github.com/ldapjs/node-ldapjs/tree/bec2ff8e7399155ebdcbf86eec2077a792b8510b/test-integration/client), e.g. [issue-940.test.js](https://github.com/ldapjs/node-ldapjs/blob/bec2ff8e7399155ebdcbf86eec2077a792b8510b/test-integration/client/issue-940.test.js), for examples of good MREs.

You may use a GitHub repository to host the code if it is too much to fit in a [code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) (or two).

**Comment by WanpengQian** [comment]
What is your LDAP server?
